docs/RELEASE_WORKFLOW.md

```diff
@@ -1,36 +1,59 @@
-# Release Workflow
+# How to do releases
+
+## Test releases
+
+* Before every official release we make a test release, similar to "Release
+  Candidates"
+* Test releases are published to supporters and the userbase through our
+  community channels, but not throught he official app stores
+* Create a forum post to inform about the test release, mention what this
+  release is about and what should be tested, here is an
+  [example](https://support.delta.chat/t/help-testing-the-upcoming-1-41-x-release/2793)
+* The goal is to do more broad user testing, looking for issues which came up
+  and gathering feedback
+* Usually after a round of feedback and bug reports we do another test release
+  and repeat, until the release stabilized
+* Optional: We can use flatpack for test releases as well (they have a
+  "testing" branch users can opt-in)
+* Test releases follow the steps described below. Official "stable" releases
+  follow them as well, but require further steps for pushing the builds into
+  the app store
 
 ## Before Releasing
 
-1. make sure no `blocker` bugs are in the issue tracker. If they are some, try to solve them first.
-
-2. Pull translations: `npm run translations-update`
-
-3. Update the local help files. (TODO: information on how do that)
+1. Make sure no `blocker` bugs are in the issue tracker. If they are some, try
+   to solve them first.
+2. Pull translations via `npm run translations-update`.
+3. Update the local help files.
 
 ## Releasing
 
-1. create a new branch for the new version (you could name it `prepare_versioncode`, for example `prepare_1.3.0`)
-
-2. update the change log (put the stuff in unreleased under a section with your new version code)
-
+1. Create a new branch for the new version (you could name it
+   `prepare_versioncode`, for example `prepare_1.3.0`)
+2. Update the change log (put the stuff in unreleased under a section with your
+   new version code)
 3. Change `version` field in `package.json` to `X.Y.Z`.
+4. Run `npm install` to update `package-lock.json`
+5. Open a PR for your branch and get it reviewed.
+6. As soon as your pr is approved / before merging: tag the latest commit with
```

Comment:
I always do it after it's merged on master to make sure everything is correct, but might also work before merging, haven't tried.

Reply:
We should probably do it _after_ merging, especially when the hash changes after a squash